src/Normalizers/Normalized/NormalizedModel.php

```diff
@@ -68,16 +73,13 @@ protected function hasModelAttribute(string $name): bool
             return $this->model->hasAttribute($name);
         }
 
-        // TODO: to use that one once we stop supporting Laravel 10
-
+        // TODO: remove this once we stop supporting Laravel 10
         if (! isset($this->attributesProperty)) {
             $this->attributesProperty = new ReflectionProperty($this->model, 'attributes');
-            $this->attributesProperty->setAccessible(true);
```

Comment:
This line does nothing because: https://www.php.net/manual/en/reflectionproperty.setaccessible.php

> Note: As of PHP 8.1.0, calling this method has no effect; all properties are accessible by default.

And we require php 8.1 in comoser.json